src/components/AppCode.vue

```diff
@@ -15,46 +26,69 @@
       </button>
 
       <div id="code" ref="code">
-        <p>
-          <span class="cname">.parent</span> {
-          <br>
-          <span class="sp">
-            <span class="ckey">display</span>:
-            <span class="cprop">grid</span>;
-          </span>
-          <br>
-          <span class="sp">
-            <span class="ckey">grid-template-columns</span>:
-            <span class="cprop">{{ colTemplate }}</span>;
-          </span>
-          <br>
-          <span class="sp">
-            <span class="ckey">grid-template-rows</span>:
-            <span class="cprop">{{ rowTemplate }}</span>;
-          </span>
-          <br>
-          <span class="sp">
-            <span class="ckey">grid-column-gap</span>:
-            <span class="cprop">{{ columngap }}px;</span>
-          </span>
-          <br>
-          <span class="sp">
-            <span class="ckey">grid-row-gap</span>:
-            <span class="cprop">{{ rowgap }}px</span>;
-          </span>
-          <br>
-          <span v-if="childarea.length > 0" class="child">
-            <span v-for="(child, i) in childarea" :key="child">
-              <span>
-                <span class="cname">.div{{ i + 1 }}</span> {
-                <span class="ckey">grid-area</span>:
-                <span class="cprop">{{ child }}</span>; }
+        <div v-if="showHtml">
+          <p>
+            <<span class="cname">div </span>
```

Comment:
It's a little weird to have the extra display tags mixed together with the real tags but I'm not sure a way around it that doesn't require reformatting everything.